daemon/emer-daemon.c

```diff
@@ -125,8 +125,6 @@ typedef struct _NetworkCallbackData
 
```

Comment:
Sorry I missed this the first time around, but we can also rm `INHIBIT_ARGS` and friends ^^.